lib/inc/internal/nodes/config_node_complex_value.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include "abstract_config_node_value.hpp"
+
+namespace hocon {
+
+    class config_node_complex_value : public abstract_config_node_value {
```

Comment:
Whoops, I did some rebasing and stuff to isolate the path code, forgot to remove this file. It's written but the tests still need things that haven't been implemented yet, so I decided not to put it up yet.